src/main/java/Duke.java

```diff
@@ -1,10 +1,109 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    final static int MAX_SIZE = 100;
+    static int idx = 0;
+    static Task[] tasks = new Task[MAX_SIZE];
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        greetUser();
+
+        serveUser();
+
+        goodbyeUser();
+
+    }
```

Comment:
I like that your main method consists of readable methods which allows readers to guess its functions.